src/main/java/org/takes/facets/flash/RsFlash.java

```diff
@@ -117,7 +111,6 @@
     /**
      * To string.
      */
-    @SuppressWarnings("unused")
     private final transient CharSequence text;
```

Comment:
@mkordas It's a public method, but I did not find any place where this toString method is used (this method was already there (lombok.ToString annotation)).